oneanddone/tests/functional/conftest.py

```diff
@@ -71,3 +76,10 @@ def task(base_url, is_local):
     if is_local:
         from oneanddone.tasks.tests import TaskFactory
         return TaskFactory.create()
+
+
+@pytest.fixture(scope='session')
```

Comment:
I see. We shouldn't make tests dependent on data they don't set up themselves though. Each test should be able to run individually, and tests should be able to execute in any order. If a test requires a tasks to already be assigned, then it it either needs to assign it via the UI, or include this as part of the task creation.